reana_server/rest/config.py

```diff
@@ -48,8 +48,9 @@ def get_config():
                 "announcement": "This is a QA instance",
                 "pooling_secs": 15,
                 "docs_url": "http://docs.reana.io/",
+                "client_pyenv": "/afs/cern.ch/user/r/reana/public/reana/bin/activate",
                 "forum_url": "https://forum.reana.io/",
-                "mattermost_url": "https://mattermost.web.cern.ch/it-dep/channels/reana",
+                "chat_url": "https://mattermost.web.cern.ch/it-dep/channels/reana",
```

Comment:
Note that `docs/opendata.json` would have to be updated too (to change `mattermost_url` there).